diff --git a/start.py b/start.py
index 347883e..448295f 100755
--- a/start.py
+++ b/start.py
@@ -18,6 +18,7 @@ import startservers
 if not startservers.start(race_detection=False):
     sys.exit(1)
 try:
+    open('/tmp/boulder-has-started','wb').write('x')
     os.wait()
 
     # If we reach here, a child died early. Log what died:
diff --git a/test/boulder-config.json b/test/boulder-config.json
index d86f017..131daae 100644
--- a/test/boulder-config.json
+++ b/test/boulder-config.json
@@ -201,7 +201,7 @@
     "issuerDomain": "happy-hacker-ca.invalid",
     "debugAddr": "localhost:8004",
     "portConfig": {
-      "httpPort": 5002,
+      "httpPort": 80,
       "httpsPort": 5001,
       "tlsPort": 5001
     },
diff --git a/test/hostname-policy.json b/test/hostname-policy.json
index a5782e2..15ad50c 100644
--- a/test/hostname-policy.json
+++ b/test/hostname-policy.json
@@ -1,13 +1,6 @@
 {
   "Blacklist": [
     "in-addr.arpa",
-    "example",
-    "example.com",
-    "example.net",
-    "example.org",
-    "invalid",
-    "local",
-    "localhost",
-    "test"
+    "invalid"
   ]
 }
diff --git a/test/rate-limit-policies.yml b/test/rate-limit-policies.yml
index ac02d19..891cb4c 100644
--- a/test/rate-limit-policies.yml
+++ b/test/rate-limit-policies.yml
@@ -4,7 +4,7 @@ totalCertificates:
   threshold: 100000
 certificatesPerName:
   window: 2160h
-  threshold: 2
+  threshold: 10000
   overrides:
     ratelimit.me: 1
     # Hostnames used by the letsencrypt client integration test.
@@ -27,7 +27,7 @@ pendingAuthorizationsPerAccount:
   threshold: 3
 certificatesPerFQDNSet:
   window: 24h
-  threshold: 5
+  threshold: 5000
   overrides:
     le.wtf: 10000
     le1.wtf: 10000
