diff --git a/cmd/policy-loader/base-rules.json b/cmd/policy-loader/base-rules.json
index a5782e2..15ad50c 100644
--- a/cmd/policy-loader/base-rules.json
+++ b/cmd/policy-loader/base-rules.json
@@ -1,13 +1,6 @@
 {
   "Blacklist": [
     "in-addr.arpa",
-    "example",
-    "example.com",
-    "example.net",
-    "example.org",
-    "invalid",
-    "local",
-    "localhost",
-    "test"
+    "invalid"
   ]
 }
diff --git a/start.py b/start.py
index 347883e..448295f 100755
--- a/start.py
+++ b/start.py
@@ -18,6 +18,7 @@ import startservers
 if not startservers.start(race_detection=False):
     sys.exit(1)
 try:
+    open('/tmp/boulder-has-started','wb').write('x')
     os.wait()
 
     # If we reach here, a child died early. Log what died:
diff --git a/test/boulder-config.json b/test/boulder-config.json
index cb3b70a..9b92ea1 100644
--- a/test/boulder-config.json
+++ b/test/boulder-config.json
@@ -161,9 +161,9 @@
     "userAgent": "boulder",
     "debugAddr": "localhost:8004",
     "portConfig": {
-      "httpPort": 5002,
-      "httpsPort": 5001,
-      "tlsPort": 5001
+      "httpPort": 80,
+      "httpsPort": 5001,
+      "tlsPort": 5001
     },
     "maxConcurrentRPCServerRequests": 16,
     "amqp": {
diff --git a/test/rate-limit-policies.yml b/test/rate-limit-policies.yml
index 05e76cf..1f1f4f8 100644
--- a/test/rate-limit-policies.yml
+++ b/test/rate-limit-policies.yml
@@ -4,7 +4,7 @@ totalCertificates:
   threshold: 100000
 certificatesPerName:
   window: 2160h
-  threshold: 2
+  threshold: 10000
   overrides:
     ratelimit.me: 1
     # Hostnames used by the letsencrypt client integration test.
@@ -22,4 +22,4 @@ registrationsPerIP:
     127.0.0.1: 1000000
 pendingAuthorizationsPerAccount:
   window: 168h # 1 week, should match pending authorization lifetime.
-  threshold: 3
+  threshold: 300
